mindsdb/integrations/mlflow_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mlflow
+mindsdb
```

Comment:
Maybe add `mindsdb` version >= to the current release to avoid using old versions.